omnigibson/transition_rules.py

```diff
@@ -63,19 +62,23 @@
 
 class TransitionRuleAPI:
     """
-    Monolithic class containing methods to check and execute arbitrary discrete state transitions within the simulator
+    Containing methods to check and execute arbitrary discrete state transitions within the simulator
     """
 
-    # Set of active rules
-    ACTIVE_RULES = set()
+    def __init__(self, scene):
+        self.scene = scene
```

Comment:
Similar to objects, do we want to break the circular dependency between scene and TransitionRuleAPI here by keeping a scene index here instead?

Reply:
The scene idx thing feels a bit brittle. Did we actually do that for objects yet or did we create a ticket for it? Lets add this comment to that ticket so that we can discuss it later. I'm not sure I agree with it.

Reply:
We did create a ticket for it. Yeah we can discuss further.